acapi/resources/backup.py

```diff
@@ -51,10 +53,29 @@ def download(self, target_file):
         url_parts._replace(query='')
         uri = url_parts.geturl()
 
-        content = self.request(uri=uri, params=query, decode_json=False)
-
-        backup_file = open(target_file, 'wb')
-        backup_file.write(content)
-        backup_file.close()
+        if stream:
+            bytes_total = bytes_read = 0
```

Comment:
Please add a comment about why this approach is needed. Someone reading the code may think it is unnecessarily complex.